ccd-definition/ComplexTypes.json

```diff
@@ -1278,5 +1278,14 @@
     "FieldType": "TextArea",
     "ElementLabel": "Directions and interim orders, if needed",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
```

Comment:
Please get rid off complex type as we discussed.

Reply:
Complex type must be there for MultiSelectList